WebUI/generate.sh

```diff
@@ -53,7 +52,7 @@ pip install -q -r requirements.txt
 echo -e "Installation successful\n"
 
 URL_SPLIT=(${REPO//// })
-AUTHOR=${AUTHOR:-${URL_SPLIT[2]}}
+AUTHOR=$(git --no-pager show -s --format='%an <%ae>' HEAD)
```

Comment:
#120  mentioned to use AUTHOR from PR, that's why I used. I'll change now. Thanks